packages/remix-dev/compiler/compiler.ts

```diff
@@ -69,7 +73,7 @@ export let create = async (ctx: Context): Promise<Compiler> => {
 
     // css compilation
     let css = await tasks.css;
-    if (!css.ok) throw css.error;
+    if (!css.ok) throw error ?? css.error;
```

Comment:
Can this just be `if (error) throw error`? The error is always assigned, so if the thing you awaited above errors, it will be there, and if anything else errors, also throw. Gets us out of here quicker and avoids future work if we something at all fails.